lib/logr.rb

```diff
@@ -0,0 +1,45 @@
+require "logger"
+
+require "logr/version"
+require "logr/entry"
+require "logr/json_formatter"
+
+module Logr
+  class Logger
+    def initialize(name, level=::Logger::DEBUG, log_device=STDOUT)
+      @logger = ::Logger.new(log_device).tap do |logger|
```

Comment:
The constructor argument is called `logdev` in core Ruby. We can follow that naming so that people who are familiar with the core stdlib know what `logdev: STDOUT` stands for.